### PR TITLE
IGAPP-869: Tunews detailsview padding

### PR DESCRIPTION
**Pull request by f1sh1918**
Please test native & web(mobile) the figures containing the images

Example Article: http://localhost:9000/testumgebung/de/news/tu-news/471

Tested on web(chrome), ios & android

### CODE REVIEW

File: build-configs/common/theme/fonts.ts [hunk 1]
```diff
@@ -25,5 +26,6 @@ export const commonFonts = {
   contentFontSize: '0.95rem',
   contentLineHeight: 1.4,
   standardParagraphMargin: '0.75rem',
-  subTitleFontSize: '1.8rem'
+  subTitleFontSize: '1.8rem',
+  figCaptionFontSize: '0.85rem'
```
Comment (f1sh1918) [inline build-configs/common/theme/fonts.ts:30]: is just a suggested font-size that is a little bit smaller the the `contentFontSize`

File: native/src/components/NativeHtml.tsx [hunk 2]
```diff
@@ -86,6 +86,16 @@ const NativeHtml = React.memo(({ content, cacheDictionary, language }: PropsType
         direction: config.hasRTLScript(language) ? 'rtl' : 'ltr'
       }}
       systemFonts={fonts}
+      tagsStyles={{
+        figure: {
+          margin: 0,
+          marginBottom: 15,
+          maxWidth: '100%',
```
Comment (f1sh1918) [inline native/src/components/NativeHtml.tsx:93]: couldn't override inline style of the width that comes from the `content` thats why i set the `maxWidth: 100%`

**Review by SteffiStuffel — APPROVED** [review]
Tested on chrome and android emulator with a tünews that was previously broken, looks good now.

File: build-configs/common/theme/fonts.ts (hunk 1 above)
Comment (klinzo) [reply build-configs/common/theme/fonts.ts:30]: I'd make this more general, something like `hintFontSize` or similar

File: web/src/components/RemoteContent.tsx [hunk 3]
```diff
@@ -8,19 +8,34 @@ const SandBox = styled.div<{ centered: boolean }>`
...
+    text-align: center;
+    margin: 15px auto;
+    @media only screen and (max-width: 640px) {
+      width: 100% !important;
+    }
+  }
+
+  & figcaption {
+    font-size: ${props => props.theme.fonts.figCaptionFontSize};
+    font-style: italic;
```
Comment (klinzo) [inline web/src/components/RemoteContent.tsx:35]: Any specific reason why you're differing in the styles on web and native?

**Review by klinzo — APPROVED** [review]
I think a little bit more padding on native would be good:
Screenshot_20220125-123455_IntegreatTestCms

File: web/src/components/RemoteContent.tsx (hunk 3 above)
Comment (f1sh1918) [reply web/src/components/RemoteContent.tsx:36]: no just forgot to set it italic on native. thx for the hint

**Pull request merged**